src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -182,18 +182,33 @@ object ArtifactOrProject {
   }
 }
 
-case class Sha1Value(toHex: String)
+sealed trait DigestType {
+  def getDigestInstance: MessageDigest
+  def expectedLength: Int
+}
+object DigestType {
+  case object Sha1 extends DigestType {
+    def getDigestInstance: MessageDigest = MessageDigest.getInstance("SHA-1")
```

Comment:
can we add `def name: String` and this can have `def name = "SHA-1"` so we can use the name in logs below?